.eslintrc.js

```diff
@@ -1,12 +1,22 @@
 module.exports = {
-  extends: ['@storybook/eslint-config-storybook'],
+  extends: ['plugin:@typescript-eslint/recommended', 'plugin:storybook/recommended'],
+  plugins: ['import'],
+  rules: {
+    '@typescript-eslint/no-explicit-any': 'off',
+  },
+  settings: {
+    'import/resolver': {
+      node: {
+        extensions: ['.js', '.ts', '.tsx', '.mjs', '.d.ts'],
+        paths: ['node_modules/', 'node_modules/@types/'],
+      },
+    },
+  },
```

Comment:
All of this (minus the `plugin:storybook/recommended` which is the new storybook ESLint config) is to account for the fact that we're not using `@storybook/eslint-config-storybook` anymore. 

Internally `@storybook/eslint-config-storybook` was doing all this and much more.

Same goes for the other 3 eslintrc files as well.

Reply:
Really good point. I think we should, since we had to update this in 4 places and the same applies whenever we upgrade to ESLint v9. I'll see if there's a simple way!

Reply:
@tmeasday we can do this with a [shared ESLint configuration](https://eslint.org/docs/v8.x/extend/shareable-configs), but adding that would bloat this PR. Are you OK if we wait to tackle the config-reuse for whenever we upgrade to ESLint v9 (which has a new config file format)?